examples/utils/bundle-size-tests/src/fluidFramework.ts

```diff
@@ -3,10 +3,8 @@
  * Licensed under the MIT License.
  */
 
-import { SharedMap, SharedTree } from "fluid-framework";
+import { SharedTree } from "fluid-framework";
 
 export function apisToBundle() {
-	SharedMap.getFactory();
-	SharedMap.create({} as any);
 	SharedTree.getFactory();
 }
```

Comment:
SharedMap is no longer part of 'fluid-framework' package.  See change description for explanation.